sampleappjava/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,20 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity
+            android:name="com.virtusize.android.auth.views.VitrusizeAuthActivity"
+            android:exported="true"
+            android:launchMode="singleTask">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:host="sns-auth"
+                    android:scheme="com.virtusize.sampleappjava.virtusize" />
+            </intent-filter>
+        </activity>
```

Comment:
Clients have to include this into their AndroidMainfest.xml to handle deep links from the SNS (Facebook) auth page loaded in a Custom Custom Tab back into the app

Reply:
@akueisara can you mention those in the README?

Reply:
@zr0z Yes, I'm working on writing the README for the Android & iOS SDKs